abstract-factory/README.md

```diff
@@ -210,7 +210,7 @@ Example use cases
 
 ## Consequences
 
-* Dependency injection in java hides the service class dependencies that can lead to runtime errors that would have been caught at compile time.
+* Dependency injection in java hides the service class dependencies that can lead to runtime notificationErrors that would have been caught at compile time.
```

Comment:
It seems that some text has been accidentally replaced in many other patterns. Please revert the changes.